modules/account-lib/src/coin/hbar/keyPair.ts

```diff
@@ -0,0 +1,50 @@
+import * as hex from '@stablelib/hex';
+import { Ed25519PrivateKey, Ed25519PublicKey } from '@hashgraph/sdk';
+import { Ed25519KeyPair } from '../baseCoin/ed25519KeyPair';
+import { KeyPairOptions, ByteKeys } from '../baseCoin/iface';
+import { NotImplementedError } from '../baseCoin/errors';
+
+export class KeyPair extends Ed25519KeyPair {
+  /**
+   * Public constructor. By default, creates a key pair with a random master seed.
+   *
+   * @param { KeyPairOptions } source Either a master seed, a private key, or a public key
+   */
+  constructor(source?: KeyPairOptions) {
+    super(source);
+  }
+
+  /**
+   * Hedera default keys format is a pair of Uint8Array keys
+   *
+   * @returns { ByteKeys } The keys in the protocol default key format
+   */
+  getKeys(): ByteKeys {
+    const result: ByteKeys = {
+      pub: hex.decode(this.keyPair.pub),

```

Comment:
Can't we just use `Buffer.from(this.keyPair.pub)` or some other built-in method?

Reply:
Oh sure, I was just using that library because it was the one used by the hashgraphSDK, and this way it's compatible with the Transaction building done for Wallet Init. We could change it and check later if it's compatible.

Reply:
or rather Gonzalo is